demo-supply-chain.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: page
+title: Reacting to a supply chain incident
+permalink: /demo-supply-chain/
+parent: GUAC demos
+nav_order: 1
+---
+
+# Reacting to a supply chain incident
+
+The next high-profile vulnerability has landed. How should you react to it? How
+do you discover which of my products and software is vulnerable? And how should
+you go about in my organization to remediate the problem? What is the patch
+plan?
+
+GUAC's GraphQL API exposes the necessary information to be able to discover how
+an organization's software catalog is affected and provides the ability to
+remediate against large-scale security incidents in a timely manner.
+
+In this demo, we will simulate the discovery of a high-profile vulnerability and
+show how you can discover what software needs to be reviewed or patched.
+CertifyBad/CertifyGood in the future will be similar to a binary authorization,
+where certain checks or policies have determined that an artifact should be
+utilized or not.
+
```

Comment:
Do you want to include a table of contents in the demos showing the section headings, as in the setup pages?